File: RemsNG/RemsNG-UI/src/app/domain/models/domain.model.ts
export class DomainModel {
    id: string = '';
    domainName: string = '';
    domainCode: string = '';
    domainStatus: string = '';
    datecreated: string = '';
    stateId: string = '';
    isLoading: boolean = false;
    errClass: string[] = new Array<string>(1);
    msg: string = '';
    isErrMsg: boolean = false;
    eventType: string = '';
}

File: RemsNG.Common/Models/DomainModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace RemsNG.Common.Models
{
    public class DomainModel
    {
        public Guid Id { get; set; }
        public string DomainName { get; set; }
        public string DomainCode { get; set; }
        public DateTime Datecreated { get; set; }
        public Guid? AddressId { get; set; }
        public string DomainStatus { get; set; }
        public string DomainType { get; set; }
        public Guid? StateId { get; set; }
    }
}
